internal/ackhandler/sent_packet_handler.go

```diff
@@ -56,17 +47,13 @@ type sentPacketHandler struct {
 	rttStats   *congestion.RTTStats
 
 	handshakeComplete bool
+
 	// The number of times the crypto packets have been retransmitted without receiving an ack.
 	cryptoCount uint32
-
-	// The number of times a TLP has been sent without receiving an ack.
-	tlpCount uint32
-	allowTLP bool
-
-	// The number of times an RTO has been sent without receiving an ack.
-	rtoCount uint32
+	// The number of times a PTO has been sent without receiving an ack.
+	ptoCount uint32
```

Comment:
Can you rename this and the following var to make their semantics more clear?

ptoCount and numPTOs next to each other are hard to understand.

Reply:
Done.